### ISSUE TITLE
Add esp32 support

### ISSUE THREAD

**Issue by alba-ado** [photonicfusion/nI2C#2]
This is a feature request. Can you add an esp32 support to this library ? :)

**Comment by srepmub** [comment]
I think you forgot to thank nitacku or his great work, and to add 'please' to your sentence.. :P

**Comment by alba-ado** [comment]
Yes, of course. I was in a rush for some reason when I wrote this. I love open source projects, and thanks a lot for this library @nitacku 

It would be very nice if you could do this, since esp32 wire calls are pretty bad. Pretty Please :)

**Comment by srepmub** [comment]
thanks! I also apologize for my moralizing message (I was also in a hurry.. P)
